include/Page/Post.php

```diff
@@ -14,6 +14,7 @@ protected function initialize() {
 		$this->page_title      = __( 'Bulk Move Posts', 'bulk-move' );
 		$this->menu_title      = __( 'Bulk Move Posts', 'bulk-move' );
 		$this->warning_message = __( 'WARNING: Posts moved once cannot be reverted. Use with caution.', 'bulk-move' );
+
```

Comment:
I am seeing lot of blank lines getting added everywhere. Let's avoid it. It is going to generate noise in diffs

Reply:
> I am seeing lot of blank lines getting added everywhere. Let's avoid it. It is going to generate noise in diffs

Fixed.